lib/run.js

```diff
@@ -21,7 +30,11 @@ function runAsyncBase(cmd, args, options = {}) {
         err.messageOnly = true;
         return reject(err);
       }
-      return resolve();
+      if (captureStdout === 'lines') {
+        stdout = stdout.split(/\r?\n/g);
```

Comment:
Could we possibly instead run `stdout.trim().split(...)`? (to avoid the check below)

Reply:
I actually had written that first, but there might be cases in which whitespace occurs intentionally at the beginning of the first line/end of the last line? :woman_shrugging: With `git` it’s not an issue,  I think, but this would also be used to spawn other programs

Reply:
:thinking: that's true but I can't come up with an example where one would use `'lines'` but would require leading/trailing spaces to be preserved.
Anyway, just a suggestion/Nit.